posawesome/posawesome/api/purchase_orders.py

```
@@ -303,73 +303,91 @@ def get_supplier_info(supplier):
 
 
 @frappe.whitelist()
-def get_last_buying_rate(supplier, item_codes):
-    """Get the last buying rate for items from the price list and recent Purchase Invoices."""
+def get_last_buying_rate(supplier, item_codes, company=None):
+    """Get the last buying rate for items from supplier price lists or recent Purchase Invoices."""
     if isinstance(item_codes, str):
         try:
             item_codes = json.loads(item_codes)
         except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)